app/views/pending_responses/index.html.haml

```diff
@@ -0,0 +1,27 @@
+-if user_signed_in?
```

Comment:
Instead of redirecting the user in the view, this should be moved to a before action in a controller.